xblock/fields.py

```diff
@@ -339,6 +339,13 @@ def default(self):
         else:
             return self._default
 
+    @staticmethod
+    def needs_name(field):
+        """
+        Returns whether the given ) is yet to be named.
+        """
+        return not field.__name__
+
```

Comment:
This was copied in from Nameable, but I recall now that I've refactored _AutoNameFieldsMetaclass such that this is now unused. I'll remove it before merging.

```suggestion
```